contracts/stride-liquid-staker/src/error.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
probably best to combine errors with the persistence lser here, maybe via macros